origami/client.py

```diff
@@ -546,10 +551,18 @@ async def subscribe_channel(self, channel: str, timeout: float):
         await self.send_rtu_request(req)
         return await asyncio.wait_for(tracker.next_trigger, timeout)
 
-    def files_channel(self, file_id):
+    @staticmethod
+    def files_channel(file_id):
         """Helper to build file channel names from file ids"""
         return f"files/{file_id}"
 
+    @staticmethod
+    def kernels_channel(file_id: UUID, user_id: UUID = None):
+        """Helper to build kernels channel name from file id"""
+        file_id_part = file_id.hex[:20]
+        user_id_part = f"-{user_id.hex[:20]}" if user_id else ""  # For playground mode only
+        return f"kernels/notebook-kernel-{file_id_part}{user_id_part}"[:63]
```

Comment:
would be nice to not generate this here, don't we automatically subscribe to the kernel channel?

Reply:
Agreed! Switched it over to `files` channel subscription instead (which automatically subscribed to the kernels channel).

Reply:
To register the callback against the RTU messages, we need to know the channel, and to construct the kernels channel name, we need this piece of code :( 

This stuff:

```
            kernel_status_tracker = self.register_message_callback(
                _kernel_status_callback,
                channel=???, # Here
                message_type="kernel_status_update_event",
                once=False,
            )
```